pkg/printers/internalversion/describe.go

```diff
@@ -2893,6 +2893,9 @@ func printTolerationsMultilineWithIndent(w *PrefixWriter, initialIndent, title,
 				if len(toleration.Effect) != 0 {
 					w.Write(LEVEL_0, ":%s", toleration.Effect)
 				}
+				if toleration.TolerationSeconds != nil {
+					w.Write(LEVEL_0, " for %ds", *toleration.TolerationSeconds)
+				}
```

Comment:
Comes from the sentence we usually describe a toleration --"tolerate taint key3=value3:NoExecute for 300s".